belt/src/services/utils.ts

```diff
@@ -21,10 +23,12 @@ export function getContractDirs(conf: config.App) {
 export function getArtifactDirs(conf: config.App) {
   const artifactDirs = ls(conf.artifactsDir)
 
-  return artifactDirs.map((d) => ({
-    dir: d,
-    version: conf.compilerSettings.versions[d],
-  }))
+  return artifactDirs
+    .map((d) => ({
+      dir: d,
+      version: conf.compilerSettings.versions[d],
+    }))
+    .filter((p) => !!p.version)
```

Comment:
When will there not be a version?

Reply:
Same as above. When you don't define it in your config file, but have it on the file system.